experimenter/query.py

```diff
@@ -0,0 +1,147 @@
+from elasticsearch import Elasticsearch
+from elasticsearch_dsl import Search, Q
+from experimenter.utils import get_problem_path, get_dataset_doc_path, get_pipelines_from_d3m
+from d3m.runtime import _get_data_and_scoring_params_from_pipeline_run as _data_score_params
+
+
+HOST = 'https://metalearning.datadrivendiscovery.org/es'
+CONNECTION = Elasticsearch(hosts=[HOST], timeout=300)
+
+         
+def get_search_query(arguments: dict = None, connection = CONNECTION, index='pipelines'):
+    index_search = Search(using=CONNECTION, index=index)
+    if arguments['id'] is not None:
+        index_search = index_search.query('match', id=arguments['id'])
+    if arguments['submitter'] is not None:
+        index_search = index_search.query('match', _submitter=arguments['submitter'])
+    return index_search
+
+
+def query_on_seeds(pipeline_id: str=None, limit: int=None, submitter: str='byu'):
```

Comment:
This query is specific to the modify generator. Let's move this code to that module or class.